### PR TITLE
fix(tabs): do not focus tabs when selectedIndex is set

### LINKED ISSUE #715: [tabs] select a tab without focusing it
I would like to select a tab without focusing it programmatically.

It's not possible due to this line:

https://github.com/ing-bank/lion/blob/0ff4199f1ff4dead0521019948c44d29cbd8a998/packages/tabs/src/LionTabs.js#L40

Could we have a method to select a tab without focusing it?

### PR DESCRIPTION
**Pull request by MathieuPuech**
Closes #715

### CODE REVIEW

File: packages/tabs/src/LionTabs.js [hunk 1]
```diff
@@ -187,34 +186,41 @@ export class LionTabs extends LitElement {
...
-    this.__firstUpdate = false;
     const stale = this.__selectedIndex;
     this.__selectedIndex = value;
-    this.__updateSelected();
+    this.__updateSelected(false);
+    this.dispatchEvent(new Event('selected-changed'));
+    this.requestUpdate('selectedIndex', stale);
+  }
+
+  __setSelectedIndex(value) {
```
[the remark this answers is not in the document]
Comment (jorenbroekema) [reply packages/tabs/src/LionTabs.js:220]: I had some discussion about this in the team and updated my comment to reflect it :)

File: packages/tabs/src/LionTabs.js [hunk 2]
```diff
@@ -34,9 +34,9 @@ const cleanButton = ({ element, clickHandler, keydownHandler, keyupHandler }) =>
   element.removeEventListener('keydown', keydownHandler);
 };
 
-const selectButton = (element, firstUpdate = false) => {
+const selectButton = (element, withFocus = false) => {
   // Don't focus on first update, as the component might be lower on the page
```
[the remark this answers is not in the document]
Comment (MathieuPuech) [reply packages/tabs/src/LionTabs.js]: I suppose the last comment was the one saying we should have a protected method `_setSelectedIndexWithFocus`

File: packages/tabs/src/LionTabs.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (MathieuPuech) [reply packages/tabs/src/LionTabs.js]: It seems to be good for me too, I made the changes

**Comment by jorenbroekema** [comment]
Looks great to me, not sure what's happening with circle-ci and karma :( seems like it's timing out a bit..

**Comment by jorenbroekema** [comment]
Ah there we go it works, thanks for this one!! 🤗

**Pull request merged**